xml/System.Formats.Tar/TarWriter.xml

```diff
@@ -44,23 +87,23 @@
         <param name="leaveOpen">
```

Comment:
Note to self: In the next docs porting PR, I need to remove the obsolete remarks text in line 86.